### PR TITLE
New parameters of climate_ir_lg

### PR DESCRIPTION
**Pull request by mhentschke**
Added parameters for the new pull request in esphome main repo

## Description:

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#1470

## Checklist:

  - [x] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [x] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

File: components/climate/ir_climate.rst [hunk 1]
```diff
@@ -120,6 +120,17 @@ Additional configuration is available for this model
   * ``MODEL_DG11J1_3A``: Temperature range is from 18 to 32 (default)
   * ``MODEL_DG11J1_91``: Temperature range is from 16 to 30
 
+LG
+---------
+
+Additional configuration is available for this platform
+
+- **header_high** (*Optional*, int): time in us for the high part of the header for the LG protocol. Defaults to 8000
```
Comment (glmnet) [inline components/climate/ir_climate.rst]: please see other components how the time unit is specified, and expand the config for all 5 parameters

**Comment by glmnet** [comment]
this needs to be created against next branch, not current

File: components/climate/ir_climate.rst (hunk 1 above)
Comment (mhentschke) [reply components/climate/ir_climate.rst]: Can you please point me to one component from which I can grab an example?

**Comment by glmnet** [comment]
btw, since you're here it'd be nice if you can add the config you use in the docs, so people can try that right away in case the defaults doesn't work for them.

**Comment by mhentschke** [comment]
> 
> 
> btw, since you're here it'd be nice if you can add the config you use in the docs, so people can try that right away in case the defaults doesn't work for them.

**Comment by glmnet** [comment]
to fix the conflicts you need to rebase this, `git rebase next` if you don't want to deal with it, you can just close this and open a new one

**Comment by mhentschke** [comment]
Sure, I'm working on the time thing and as soon as I finish that, I'll add an example.

File: components/climate/ir_climate.rst (hunk 1 above)
Comment (mhentschke) [reply components/climate/ir_climate.rst]: Done, I think I'm now using the correct time standard for this.

**Comment by mhentschke** [comment]
Just put a code snippet of my config and also corrected the Time settings to match the code PR

**Comment by mhentschke** [comment]
> 
> 
> to fix the conflicts you need to rebase this, `git rebase next` if you don't want to deal with it, you can just close this and open a new one

Just saw this. Working on it

**Comment by mhentschke** [comment]
Should be good now. Sorry for the trouble. First PR here, still learning

**Comment by glmnet** [comment]
No it's not... it will be good when up there in "files changes" there is only 1 file change

**Comment by mhentschke** [comment]
I did what you told me. I ran "git rebase origin/next" and then solved all the conflicts accepting the changes from origin/next. Not sure what went wrong or how to fix it.

**Comment by mhentschke** [comment]
esphome/esphome-docs#963 should solve the issue

**Comment by glmnet** [comment]
thanks, I'm sorry the rebase didn't work.

**Pull request closed without merging**